main.ts

```diff
@@ -424,76 +445,103 @@ class ColoredTagsPluginSettingTab extends PluginSettingTab {
 
 		containerEl.empty();
 
-		new Setting(containerEl)
-			.setName('Palette size')
-			.setDesc('How many different colors are available.')
-			.addSlider(slider =>
-				slider.setLimits(8, 32, 8)
-					.setValue(this.plugin.settings.palette)
-					.onChange(async (value) => {
-						slider.showTooltip();
-						this.plugin.settings.palette = value;
-						await this.plugin.saveSettings();
-						this.renderPalette(paletteEl);
-					})
-			)
-
 		const paletteEl = containerEl.createEl("div", {
 			cls: "palette",
 			attr: {style: `display: flex; align-items: stretch`}
 		});
 		this.renderPalette(paletteEl);
 
 		new Setting(containerEl)
-			.setName('Palette shift')
-			.setDesc('If the colors of some tags don\'t fit, you can shift the palette.')
-			.addSlider(slider =>
-				slider.setLimits(0, 10, 1)
-					.setValue(this.plugin.settings.seed)
-					.onChange(async (value) => {
-						slider.showTooltip();
-						this.plugin.settings.seed = value;
-						await this.plugin.saveSettings();
-						this.renderPalette(paletteEl);
-					})
-			)
-
-		new Setting(containerEl)
-			.setName('Saturation')
-			.addDropdown(dropdown =>
-				dropdown.addOption(String(DEFAULT_SETTINGS.chroma), 'Default')
-					.addOptions({
-						'5': 'Faded',
-						'32': 'Moderate',
-						'64': 'Vivid',
-						'128': 'Excessive',
-					})
-					.setValue(String(this.plugin.settings.chroma))
-					.onChange(async (value) => {
-						this.plugin.settings.chroma = Number(value);
-						await this.plugin.saveSettings();
-						this.renderPalette(paletteEl);
-					})
-			)
+            .setName('Enable custom colors')
+            .setDesc('Ignore other pallet settings and use only custom colors.')
+            .addToggle(toggle => toggle
+                .setValue(this.plugin.settings.enableCustomColors)
+                .onChange(async (value) => {
+                    this.plugin.settings.enableCustomColors = value;
+                    await this.plugin.saveSettings();
+					this.display();
+					this.renderPalette(paletteEl);
+                }));
+
+		if(this.plugin.settings.enableCustomColors !== true) {
```

Comment:
This is unusual condition. Maybe `if (!this.plugin.settings.enableCustomColors)` will be better?